src/lib/mdns/Advertiser_ImplMinimalMdns.cpp

```diff
@@ -569,6 +584,114 @@ FullQName AdvertiserMinMdns::GetCommisioningTextEntries(const CommissionAdvertis
     }
 }
 
+bool CanAdvertiseOn(chip::Inet::InterfaceAddressIterator & current)
+{
+    if (!current.IsUp())
+    {
+        return false;
+    }
+
+    if (!current.SupportsMulticast())
+    {
+        return false;
+    }
+
+    char name[64];
```

Comment:
Can we perhaps size this intelligently to `IF_NAMESIZE`?  It seems like that would be preferable to hard-coding a constant that will either be needlessly large, or unexpectedly fail at runtime on some exotic target.

Reply:
It looks like in Zephyr this can be up to `Z_DEVICE_MAX_NAME_LEN` characters.  To avoid use of this inline constant (which I still advocate we avoid), some sort of platform-mapped macro might be needed.